src/main/java/au/org/emii/ncdfgenerator/NcdfGenerator.java

```diff
@@ -0,0 +1,60 @@
+
+package au.org.emii.ncdfgenerator;
+
+import au.org.emii.ncdfgenerator.cql.ExprParser;
+import au.org.emii.ncdfgenerator.cql.IExprParser;
+import au.org.emii.ncdfgenerator.cql.IDialectTranslate;
+import au.org.emii.ncdfgenerator.cql.PGDialectTranslate;
+
+import java.sql.Connection;
+import java.io.InputStream;
+import java.io.FileInputStream;
+import java.io.OutputStream;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+
+public class NcdfGenerator
+{
+	final IExprParser parser;
+	final IDialectTranslate translate;
+	final ICreateWritable createWritable;
+	final String layerConfigDir;
+
+	public NcdfGenerator( String layerConfigDir, String tmpCreationDir  )
```

Comment:
This lib is using a temp directory to create the zip file.  Can it take care of managing that itself, rather than the calling code having to worry about it?